_posts/2021-12-20-start-a-workspace-by-entering-a-url.adoc

```diff
@@ -18,7 +21,7 @@ As a user, you can create a workspace containing a clone of a Git repository by
 
 === Short URL syntax for cloning hosted Git repositories
```

Comment:
```suggestion
=== Defining the Git repository to clone
```

Reply:
Question: I understand we can not clone multiple repositories, but it is not clearly stated.